components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/api/APIStore.java

```diff
@@ -242,5 +242,37 @@ SubscriptionResponse addApiSubscription(String apiId, String applicationId, Stri
      */
     List<Rating> getUserRatingDTOList(String apiId) throws APIManagementException;
 
+    /**
+     * Add comment for an API
+     *
+     * @param comment the comment text
+     * @param apiId UUID of the API
+     * @return String UUID of the created comment
+     */
```

Comment:
@throws is missing in comments. Check the other comments as well. It is missing in multipleplaces